.github/workflows/ciwheel.yml

```diff
@@ -24,11 +24,14 @@ jobs:
         with:
           fetch-depth: 0
           submodules: recursive
+      - name: Create unique VERSION
+        if: endsWith(github.ref,'ciwheel') || endsWith(github.ref,'rc') # https://docs.github.com/en/actions/learn-github-actions/expressions
+        run: python3 -c 'import time;f=open("VERSION","r+");content = f.readlines();f.seek(0);f.write(content[0].strip()+time.strftime("%Y%m%d%H%I%S"))'
       - name: Install cibuildwheel
         run: python3 -m pip install cibuildwheel
       - name: Build wheels
         run: python3 -m cibuildwheel --output-dir dist
-      - uses: actions/upload-artifact@v2
+      - uses: actions/upload-artifact@v3
```

Comment:
Nope, they don't overwrite, but append. So, when the artifacts are collected in the following step, they appear all together, which is convenient, because they all need to be uploaded. The filenames are unique, they follow some PEP number.